js/index.js

```diff
@@ -18,6 +18,7 @@ let updatesEnabled = false;
 
 type GeoConfiguration = {
   skipPermissionRequests: boolean,
+  authorizationLevel: 'always' | 'whenInUse',
```

Comment:
Can we add an `'auto'` value just incase someone wants to set the default behaviour explicitly in a type safe way.

```suggestion
  authorizationLevel: 'always' | 'whenInUse' | 'auto',
```